frontend/src/pages/ui/Sidebar.tsx

```diff
@@ -34,12 +32,14 @@ export default function Sidebar(props: SidebarProps) {
 
   return (
     <>
-      <div className={styles.StickySidebarBox} style={{ top: tocOffset }}>
-        <Card className={styles.SidebarTopicInfoButtonBox} raised={true}>
+      {/* @BEN: is it acceptable to combine tw utility classes and inline style={{}} prop (for dynamic vals)? */}
+      <div className='sticky' style={{ top: tocOffset }}>
+        <div className='rounded-sm bg-white shadow-raised md:m-cardGutter md:flex md:w-[90%] md:flex-col md:justify-center md:p-2'>
           <TopicInfoModalButton />
-        </Card>
+        </div>
+
         <div className='mode-selector-box'>
-          <Card raised={true} className={styles.SidebarModeSelectorBox}>
+          <div className='rounded-sm bg-white shadow-raised md:m-cardGutter md:flex md:w-[90%] md:flex-col md:justify-center md:px-2 md:py-4'>
```

Comment:
we actually have a 90% width defined in the tailwind.config as it's used elsewhere, can you swap it out for `md:w-[90%]` `md:w-90p` ? The idea is that once we have gone through the whole app, we can see in the config which percentage widths and other design tokens we are using, and eventually standardize them for more visual consistency.

Reply:
ah! corrected